Documentation/General/finalReport/overview.tex

```diff
@@ -1 +1,40 @@
-\chapter{Overview of the developed and implemented software product}
\ No newline at end of file
+\chapter{Overview of the developed and implemented software product}
+
+\section{Introduction}
+When entering MoodCat, a user can choose multiple of a list of moods to listen music to. Based on these moods a list of rooms
```

Comment:
"When entering MoodCat, a user can choose multiple of a list of moods to listen music to" doesn't sound right.
Maybe "When entering MoodCat, a user can choose one or multiple moods from a list of moods to listen to.", altough I'm not totally pleased with that either, can't think of something better.